src/PgManyToManyRelationPlugin.ts

```diff
@@ -1,15 +1,18 @@
 import type { PgResource, PgSelectSingleStep } from "@dataplan/pg";
 import type {} from "graphile-config";
 import type { GraphQLObjectType } from "graphql";
+import type { SQL } from "pg-sql2";
 import type {} from "postgraphile";
+import type { PgManyToManyRelationDetails, PgTableResource } from ".";
 import createManyToManyConnectionType from "./createManyToManyConnectionType";
 import manyToManyRelationships from "./manyToManyRelationships";
-import { PgManyToManyRelationDetails, PgTableResource } from ".";
-import type { SQL } from "pg-sql2";
-
+import { EXPORTABLE } from "graphile-build";
 const version = require("../package.json").version;
 
-export const junctionSymbol = Symbol("junction");
+export const junctionSymbolContainer = EXPORTABLE(
+  () => Object.assign({}, { symbol: Symbol("junction") }),
```

Comment:
I don't think there's any benefit to using Object.assign here?
```suggestion
  () => ({ symbol: Symbol("junction") }),
```